Regression/Checksum/benchmarks_json/BeamBeamCollision.json

```diff
@@ -1,96 +1,96 @@
 {
   "lev=0": {
-    "Bx": 970573570349.2142,
-    "By": 970493693614.8823,
-    "Bz": 18743533.08373785,
-    "Ex": 2.9100804648817213e+20,
-    "Ey": 2.910188719414779e+20,
-    "Ez": 1.309938057448976e+17,
-    "rho_beam1": 7.969171294602906e+16,
-    "rho_beam2": 7.969079911431987e+16,
-    "rho_ele1": 234837475722889.97,
-    "rho_ele2": 277285508564124.12,
-    "rho_pos1": 229824415763789.62,
-    "rho_pos2": 286513388076434.4
+    "Bx": 958613893612.4355,
+    "By": 958606286084.2804,
+    "Bz": 50291310.73170665,
+    "Ex": 2.873993867215236e+20,
+    "Ey": 2.8740263458334176e+20,
+    "Ez": 1.3469564521662371e+17,
+    "rho_beam1": 7.96926761425663e+16,
+    "rho_beam2": 7.969234189119718e+16,
+    "rho_ele1": 325562788515568.7,
+    "rho_ele2": 301373974746269.7,
+    "rho_pos1": 314013278169396.75,
+    "rho_pos2": 311298336003435.56
```

Comment:
These values changed quite drastically. Is that within your expectation?

Reply:
Other tests with changes this large had resolution changes as well.

Reply:
Yes, I think that this makes sense, because `ele1`, `ele2` and `pos1`, `pos2` come from QED pair creation processes that occur when the two beams are crossing. The rates for these QED processes are quite sensitive to the magnitude of the local E/B field, and thus it is quite possible that filtering alters the corresponding E/B in a way that already significantly modifies these QED rates.

I think that this indicates that, in order to reach numerical convergence, this simulation should be run at higher resolution.